### ISSUE TITLE
Data Dashboard: Add a dropdown option, “selection needed” as default on landing

### ISSUE THREAD

**Issue by efregoso** [SatcherInstitute/health-equity-tracker#373]
Currently, the dropdown options initialize to "COVID-19" and "the United States".
We want to allow the user to select these options on landing, rather than assigning them and having the user reassign them, so we can add a dropdown option "selection needed" and have this be the default on landing.

Alternatively, we could have the dropdown option become a blank text field that opens up dropdown options once clicked on or typed into; this can be further considered after implementation.

**Comment by aaronsn** [comment]
This seems like it should be low priority, right?

**Comment by efregoso** [comment]
Agreed -- In general, anything not marked with the "high priority" label is
considered mid-to-low priority.

**Comment by efregoso** [comment]
Completion time estimate: < 1 week

**Comment by benhammondmusic** [comment]
closed in #1782
